virtual_rainforest/__init__.py

```diff
@@ -2,6 +2,8 @@
 
 # Import all module schema here to ensure that they are added to the registry
 from virtual_rainforest.core import schema  # noqa
+from virtual_rainforest.models.abiotic import schema  # noqa
+from virtual_rainforest.models.abiotic.model import AbioticModel  # noqa
 from virtual_rainforest.models.plants import schema  # noqa
```

Comment:
I tried to add the `AbioticModel` to the list of models below but it was sorted back in here, I think it might do it alphabetically?

Reply:
Ahh that make sense, not completely sure why `isort` let me get away with that in the past